### DISCUSSION TITLE
Is it possible to use Slideouts just as an informative view, without save actions?

### DISCUSSION THREAD

**Discussion by gopeter** [craftcms/cms#15198] [Q&A] [answered]
I'm building a slideout into my plugin and want to create something like a "more details" view. But slideouts always have "Cancel" and "Save" buttons. Is there a way to customize this?

**Comment by brandonkelly** [comment, marked as answer]
Yes, you can create a slideout manually via JavaScript, and put whatever you want in them:

```js
const mySlideout = new Craft.Slideout('<p>Hello</p>');
```

For example, field layout designers will create slideouts for showing field layout element settings:

https://github.com/craftcms/cms/blob/1b2dd98384df894c87f3d6c21ad804220f63e19c/src/web/assets/cp/src/js/FieldLayoutDesigner.js#L344-L372

**Reply by gopeter** [reply]
Ah perfect, thanks a lot!
